_posts/2024-12-26-issues-with-microros.md

```diff
@@ -0,0 +1,230 @@
+---
+title: "Less is More: Why We Abandoned Micro-ROS for UART"
+date: 2024-11-29
+categories: [Firmware]
+tags: [ros, control, motors, micro-ros, communication]     # TAG names should always be lowercase
+author: sis
+---
+
+
+If you've worked on robotics projects, chances are someone has mentioned ROS (Robot Operating System). And if you're anything like me, you’re always eager to try out shiny new tools.
+
+When we started working on our Two Wheel Inverted Pendulum (TWIP) project, I was determined to use ROS—mainly because it's a that counteless robotics videos swear by. 
```

Comment:
Proposed change: I was determined to use ROS—mainly because it's a system that countless robotics videos swear by.